src/InstrumentationEngine.Attach/InstrumentationEngine.Attach.csproj

```diff
@@ -46,4 +47,8 @@
     <Error Condition="'%(XsdSchemaFile.Namespace)' == ''" Text="Missing Namespace attribute for XsdSchemaFile %(XsdSchemaFile.FullPath)" />
     <Exec Command="&quot;$(SDK40ToolsPath)\xsd.exe&quot; &quot;%(XsdSchemaFile.FullPath)&quot; /classes /namespace:%(XsdSchemaFile.Namespace) /language:%(XsdSchemaFile.Language) /out:$(MSBuildThisFileDirectory)" />
   </Target>
+  <ItemGroup>
+    <!-- dotnet core 3.1 produces both a dll and exe; project references only pick up the dll so this explicitly adds the exe for signing -->
+    <FilesToSign Include="$(OutputPath)\Microsoft.InstrumentationEngine.Attach.exe"/>
```

Comment:
This comment is confusing. I would expect the Microbuild targets file to sign the exe output as well. Whether other projects have references to this one seems orthogonal.

Reply:
Ah, the comment was copied from Aerobee which uses a signproj with a projectreference. I had assumed this was what caused the dll to get signed and not the exe, hence the comment.